packages/config/config/devices/0x010f/fgd212_3.5.json

```diff
@@ -32,62 +32,64 @@
 	"supportsZWavePlus": true,
 	"paramInformation": {
 		"1": {
-			"label": "Minimum brightness level",
-			"description": "set automatically during the calibration",
+			"label": "Minimum Brightness Level",
+			"description": "For manual override, this is set during auto-calibration",
 			"valueSize": 1,
 			"minValue": 1,
 			"maxValue": 98,
 			"defaultValue": 1
 		},
 		"2": {
-			"label": "Maximum brightness level",
-			"description": "parameter is set automatically during the calibration process",
+			"label": "Maximum Brightness Level",
+			"description": "For manual override, this is set during auto-calibration",
 			"valueSize": 1,
-			"minValue": 1,
+			"minValue": 2,
 			"maxValue": 99,
 			"defaultValue": 99
 		},
 		"3": {
-			"label": "Incandescence level of dimmable CFL*",
+			"label": "Incandescence Level of Dimmable CFL*",
 			"description": "Virtual value (percentage between 1% and 99%)",
 			"valueSize": 1,
 			"minValue": 1,
 			"maxValue": 99,
 			"defaultValue": 1
 		},
 		"4": {
-			"label": "Incandescence time of dimmable CFL*",
-			"description": "time required for switching. 0s - 25,5s",
+			"label": "Incandescence Time of Dimmable CFL*",
+			"unit": "100 ms",
```

Comment:
Nitpicking, but please move the unit to after the valueSize throughout.

Reply:
Sure. Is the order specified anywhere in the style guide? Is it required properties first and then optional?

Reply:
@blhoward2 I don't think we have this requirement written in the style guide yet. It is only there for the top level properties:
https://zwave-js.github.io/node-zwave-js/#/config-files/file-format?id=configuration-file-format